### ISSUE TITLE
Mechanism to create a page that embeds read-only versions is several whiteboards

### ISSUE THREAD

**Issue by droundy** [lovasoa/whitebophir#116]
I'd love to hear your thoughts on a way to create a page that embeds several whiteboards. The idea is for an instructor to be able to monitor several groups of students and see who might be at a stage where they need help.

**Comment by lovasoa** [comment]
The easiest is probably to use a set of iframes representing the various boards. Here is a small example I put together quickly : https://svelte.dev/repl/06cb5086951146478e918de7684adcf0?version=3.24.1

Making the boards read-only would be much more difficult. There is currently no authentication system, and thus no way to prevent someone from writing to a board to which he has access.

**Comment by droundy** [comment]
The read-only idea isn't so much about preventing modification as about enabling the boards to be smaller by removing the editing tools...

**Comment by lovasoa** [comment]
I see... As a quick hack, if you are hosting the monitor and the whiteboards on the same domain, you can do the following : 

```js
my_iframe.contentWindow.document.getElementById("menu").style.display = "none"
```

**Comment by droundy** [comment]
I'll try that. (when I find time). On a related note, does the server know what the view ports of the clients are? It would be lovely to be able to see what students are looking at, and even to automatically follow their view.

**Comment by lovasoa** [comment]
No, at the moment, clients do not send their full viewport to the server. But they send their position via the cursor tool.

**Comment by drpetersen** [comment]
This is really cool, thanks! One would also have to somehow dectivate the keyboard shortcuts, right?
I just tried your approach by setting `style="display: none"` for the menu `div` (quick and dirty, via the browser's developer tools), and found that I could still switch to the pen tool (for instance) by pressing the P key and then paint on the canvas. I fear that ultimately, someone who shouldn't might discover this …
